verify_urls.go

```diff
@@ -42,6 +42,7 @@ func main() {
         errormessage := err
         if errormessage == nil {
             errormessage = errors.New(http.StatusText(resp.StatusCode))
+            resp.Body.Close()
```

Comment:
The way we talked about privately (adding `resp.Body.Close()` after the `if`) wouldn't work with properly with unresolvable domains `https://abcdefgmozialla.org/`, so thats why I solved it that way.